index.js

```diff
@@ -20,13 +21,23 @@ function detectApiVersionMiddleware(req, res, next) {
   next();
 }
 
-function static(app, path) {
+function statics(app, path) {
   path = path || '/../../public';
 
   app.use('/', express.static(__dirname + path));
 }
 
-function errorHandler(app, logger) {
+function _translate(text, lang, translationObject) {
+  return _.get(translationObject, `${text}.${lang}`, text);
+}
+
+/**
+ * Configures the app with an error handler
+ * Config has a shape of {
+ *  translationObject: Object,
+ * }
+ */
+function errorHandler(app, logger, config) {
```

Comment:
let's move logger inside the `config` object as well. We will just release this as version 2.0.0 so that we don't break existing apps